src/state/stores.js

```diff
@@ -1,21 +1,27 @@
 import { mapValues, pickBy } from "lodash";
 import { stringify } from "query-string";
-import { get, writable } from "svelte/store";
+import { writable, get } from "svelte/store";
 
 export const pageTitle = writable("");
 export const pageState = writable({});
 
-// synchronizes url state with page state
-export const updateURLState = (push = false) => {
-  const simplifiedState = mapValues(
-    pickBy(get(pageState), (v) => (typeof v !== "string" && v) || v.length > 0),
+// updates page state and synchronizes with the url
+export const updatePageState = (newState, push = false) => {
+  const mergedState = mapValues(
+    pickBy(
+      { ...get(pageState), ...newState },
+      (v) =>
+        (typeof v !== "string" && v) ||
+        (v && ((v.length > 0 && v !== "page") || v !== "1"))
```

Comment:
Maybe you've already explained this to me before (sorry!), but why do we need these conditions here, especially this part `(v && ((v.length > 0 && v !== "page") || v !== "1"))`?

Reply:
The page thing was a silly experiment of mine to reduce the length of the URLs (there's no need to include it the URL if page=1), but we should do that in a more principled way (and not as part of this PR). I'll take it out!